osidb/serializer.py

```diff
@@ -1999,7 +2000,15 @@ class Meta:
 class FlawCollaboratorSerializer(TrackingMixinSerializer):
     """FlawCollaborator serializer"""
 
+    class LabelTypeEnum(models.TextChoices):
+        ALIAS = FlawLabel.FlawLabelType.ALIAS
+        CONTEXT_BASED = FlawLabel.FlawLabelType.CONTEXT_BASED
```

Comment:
You can just add a classmethod / class property in the original LabelTypeEnum to return a subset of choices, it's a common pattern we use throughout OSIDB